Vagrantfile

```diff
@@ -1,36 +1,29 @@
 # -*- mode: ruby -*-
 # vi: set ft=ruby :
 
-# Vagrantfile API/syntax version. Don't touch unless you know what you're doing!
-VAGRANTFILE_API_VERSION = '2'.freeze
-
 Vagrant.require_version '>= 1.5.0'
 
-Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
-  # All Vagrant configuration is done here. The most common configuration
-  # options are documented and commented below. For a complete reference,
-  # please see the online documentation at vagrantup.com.
-
-  config.vm.hostname = 'stackstorm-berkshelf'
+# Autoinstall for vagrant-omnibus plugin.
+# See https://github.com/chef/vagrant-omnibus
+unless Vagrant.has_plugin?('vagrant-omnibus')
+  system('vagrant plugin install vagrant-omnibus') || exit!
```

Comment:
Yeah, defaults to get things running fast.
My thinking that it's not ideal, but it's not possible to run this particular `Vagrantfile` without installing them. So it's balance.

If you have strong opinion that we shouldn't do that and it's kind of super crazy, - I'm OK to change that ;)